src/pages/auth/register.tsx

```diff
@@ -64,7 +107,7 @@ const Register = () => {
               >
                 <Input
                   onChange={userNameChange}
-                  errorMessage={`userName should be 2-29 characters without number or special character!`}
+                  errorMessage={`Username should be 2-29 characters without special character!`}
```

Comment:
Remember: that's not a userName. That's the User's name.

`Felipe Macedo !== felipinhomacedo77`

So we can ask like: "Oops, looks like that's not your real name."